python/graphstorm/config/argument.py

```diff
@@ -1085,6 +1086,17 @@ def label_field(self):
             "Must provide the feature name of labels through label_field"
         return self._label_field
 
+    @property
+    def semi_supervised(self):
```

Comment:
do you think if this naming is a bit confusing? we also say that GCN model training is semi-supervised.

Reply:
Yes, what name do you suggest?